packages/io/dxf-serializer/index.js

```diff
@@ -8,56 +8,48 @@ Copyright (c) 2018 Z3 Development https://github.com/z3dev
 All code released under MIT license
 
 Notes:
-1) CAG conversion to:
+1) geom2 conversion to:
      POLYLINE
      LWPOLYLINE
-2) CSG conversion to:
+2) geom3 conversion to:
      3DFACE
      POLYLINE (face mesh)
-3) Path2D conversion to:
+3) path2 conversion to:
      LWPOLYLINE
 TBD
 1) support binary output
 2) add color conversion
-3) translation for CSG
-
 */
 
-const {isCAG, isCSG} = require('@jscad/csg')
-const {ensureManifoldness} = require('@jscad/io-utils')
-const {toArray} = require('@jscad/io-utils/arrays')
-const {dxfHeaders, dxfClasses, dxfTables, dxfBlocks, dxfObjects} = require('./autocad_AC2017')
+const { geometry, utils } = require('@jscad/csg')
+
+const { ensureManifoldness } = require('@jscad/io-utils')
+
+const { dxfHeaders, dxfClasses, dxfTables, dxfBlocks, dxfObjects } = require('./autocad_AC2017')
 
 const mimeType = 'application/dxf'
 
 /** Serialize the give objects to AutoCad DXF format.
- * @param {Object} [options] - options for serialization
+ * @param {Object} options - options for serialization, REQUIRED
  * @param {Object|Array} objects - objects to serialize as DXF
  * @returns {Array} serialized contents, DXF format
  */
-const serialize = (...params) => {
-  let options = {}
-  let objects
-  if (params.length === 0) {
-    throw new Error('no arguments supplied to serialize function !')
-  } else if (params.length === 1) {
-    // assumed to be object(s)
-    objects = Array.isArray(params[0]) ? params[0] : params
-  } else if (params.length > 1) {
-    options = params[0]
-    objects = params[1]
-  }
-  // make sure we always deal with arrays of objects as inputs
-  objects = toArray(objects)
-
+const serialize = (options, ...objects) => {
   const defaults = {
     cagTo: 'lwpolyline', // or polyline
     csgTo: '3dface', // or polyline
     pathTo: 'lwpolyline',
+    entityId: 0,
```

Comment:
what does this one do ?

Reply:
Removed from defaults, as the entityId is used to control processing.